.github/workflows/build.yml

```
@@ -30,15 +30,11 @@ name: Build
 
 on: [push, pull_request]
 
-jobs:
+concurrency:
+  group: ${{ github.workflow }}-${{ (github.ref == 'refx/heads/main' && github.run_id) || github.event.pull_request.number || github.ref }}
```

```suggestion
  group: ${{ github.workflow }}-${{ (github.ref == 'refs/heads/main' && github.run_id) || github.event.pull_request.number || github.ref }}
```

Is it `refs` or `refx` ?